core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/WorkspaceManagerDAO.scala

```diff
@@ -1,11 +1,34 @@
 package org.broadinstitute.dsde.rawls.dataaccess.workspacemanager
 
+import bio.terra.profile.model.ProfileModel
 import bio.terra.workspace.client.ApiException
 import bio.terra.workspace.model._
+import com.google.common.io.BaseEncoding.base64Url
 import org.broadinstitute.dsde.rawls.model.{DataReferenceDescriptionField, DataReferenceName, RawlsRequestContext}
 import org.broadinstitute.dsde.workbench.model.{ErrorReportSource, WorkbenchEmail}
 
+import java.nio.ByteBuffer
 import java.util.UUID
+import scala.util.Try
+
+object WorkspaceManagerDAO {
+
+  /** @see bio.terra.stairway.ShortUUID#get() */
+  def decodeShortUuid(shortUuid: String): Option[UUID] =
+    Try {
+      val bytes = ByteBuffer.wrap(base64Url.decode(shortUuid))
+      val hi = bytes.getLong(0)
+      val lo = bytes.getLong(8)
+      new UUID(hi, lo)
+    }.toOption
+
+  def encodeShortUUID(uuid: UUID): String = {
+    val byteBuffer = ByteBuffer.allocate(16)
+    byteBuffer.putLong(uuid.getMostSignificantBits)
+    byteBuffer.putLong(uuid.getLeastSignificantBits)
+    base64Url().encode(byteBuffer.array())
+  }
+}
```

Comment:
I wish this wasn't necessary

Reply:
I suggest adding a comment with the ticket number (PF-1268) so we know when we can update cloneWorkspace and remove this.

Reply:
sure

Reply:
Maybe move these into the ShortUUID class itself

Reply:
Would be nice, but it's in Stairway. I don't think Rawls depends on Stairway.

Reply:
It does - at least calling ShortUUID.get works in tests